easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0-pat-6.5.1.eb

```diff
@@ -0,0 +1,48 @@
+# contributed by Luca Marsella (CSCS)
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+name = 'GROMACS'
+version = "2016.3"
+plumedversion = '2.3.0'
+cudaversion = '8.0'
+patversion = '6.5.1'
+versionsuffix = '-PLUMED-%s-cuda-%s-pat-%s' % (plumedversion, cudaversion, patversion)
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'usempi': 'True', 'openmp': 'True'}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+preconfigopts = ' pushd ../gromacs-%(version)s && plumed-patch -p --runtime --shared <<< 2 && popd && '
```

Comment:
@lucamar, I believe that the GROMACS' easyblock already does this operation, as soon as you add PLUMED in the dependency list.